### PR TITLE
Print error log when neither left is green nor right is blue

### PR DESCRIPTION
**Pull request by shrekshao**
Print detailed pixels value error when neither left is green nor right is blue to help better understanding of flakiness.

### CODE REVIEW

File: sdk/tests/conformance/textures/misc/texture-video-transparent.html
Comment (kenrussell) [inline sdk/tests/conformance/textures/misc/texture-video-transparent.html:126]: It seems to me that if the only error is that the left side isn't green, that this detail log won't show any more information.

**Review by kenrussell — APPROVED** [review]
Have you tried injecting various failures into this test (making each of the comparisons fail individually) and seeing what the error messages are? If not, please do and revise as necessary before merging. Thanks.

**Comment by kenrussell** [comment]
Note: I'd recommend printing out all of the read-back values from all of the locations very clearly in the situation where the test fails for any reason.

File: sdk/tests/conformance/textures/misc/texture-video-transparent.html
Comment (kenrussell) [inline sdk/tests/conformance/textures/misc/texture-video-transparent.html:95]: Did you mean to make this change, or was it only for testing?

Comment (shrekshao) [reply sdk/tests/conformance/textures/misc/texture-video-transparent.html:95]: that was meant to be deleted. BTW I'm extending to print the whole buf.

**Comment by shrekshao** [comment]
Now outputing whole buffer value when failed. Will merge and roll if that looks good. @kenrussell

File: sdk/tests/js/webgl-test-utils.js
Comment (kenrussell) [inline sdk/tests/js/webgl-test-utils.js:1286]: Hopefully this has no effect on other tests.

**Review by kenrussell — APPROVED** [review]
Looks good!

**Pull request merged**
